lib/livebook_web/live/session_live/export_component.ex

```diff
@@ -0,0 +1,66 @@
+defmodule LivebookWeb.SessionLive.ExportComponent do
```

Comment:
Curiosity: does this need to be a live component?

Reply:
I guess we want to take a snapshot at the moment we press export? 🤔 hence it having its own state?

Reply:
Yeah, we need to make sure to only load the content from session once.